{{cookiecutter.project_name}}/CONTRIBUTING.rst

```diff
@@ -0,0 +1,107 @@
+.. highlight:: shell
+
+============
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every little bit
+helps, and credit will always be given.
+
+You can contribute in many ways:
+
+Types of Contributions
+----------------------
+
+Report Bugs
+~~~~~~~~~~~
+
+Report bugs at https://github.com/ESSS/{{ cookiecutter.package_name }}/issues.
+
+If you are reporting a bug, please include:
+
+* Your operating system name and version.
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+Fix Bugs
+~~~~~~~~
+
+Look through the GitHub issues for bugs. Anything tagged with "bug" and "help
+wanted" is open to whoever wants to implement it.
+
+Implement Features
+~~~~~~~~~~~~~~~~~~
+
+Look through the GitHub issues for features. Anything tagged with "enhancement"
+and "help wanted" is open to whoever wants to implement it.
+
+Write Documentation
+~~~~~~~~~~~~~~~~~~~
+
+{{ cookiecutter.package_name }} could always use more documentation, whether as part of the
+official Barril docs, in docstrings, or even on the web in blog posts,
+articles, and such.
+
+Submit Feedback
+~~~~~~~~~~~~~~~
+
+The best way to send feedback is to file an issue at https://github.com/ESSS/{{ cookiecutter.package_name }}/issues.
+
+If you are proposing a feature:
+
+* Explain in detail how it would work.
+* Keep the scope as narrow as possible, to make it easier to implement.
+* Remember that this is a volunteer-driven project, and that contributions
+  are welcome :)
+
+Get Started!
+------------
+
+Ready to contribute? Here's how to set up `{{ cookiecutter.package_name }}` for local development.
+
+1. Fork the `{{ cookiecutter.package_name }}` repo on GitHub.
+2. Clone your fork locally::
+
+    $ git clone git@github.com:your_github_username_here/{{ cookiecutter.package_name }}.git
+
+3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
```

Comment:
We probably should update the instructions to use `venv` to create the virtual env:

```
    $ python -m venv .env
    $ cd {{ cookiecutter.package_name }}/
    $ pip install -e .[testing,docs]
```

We should also include instructions on how to use conda, like here: https://github.com/ESSS/barril/blob/master/README.rst#development

Reply:
✔️ Commit 330afbf update the instructions ;)